src/testhost.x86/UnitTestClient.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.TestExecutor
+{
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Microsoft.VisualStudio.TestPlatform.TestHost;
+
+    /// <summary>
+    /// Initialization point for Old UWP application
+    /// </summary>
+    public static class UnitTestClient
+    {
+        /// <summary>
+        /// Create default UI for UWP app
+        /// </summary>
+        public static void CreateDefaultUI()
+        {
+
+        }
+
+
+        /// <summary>
+        /// Entry point for testhost, in App Model(UWP) scenario
+        /// </summary>
+        /// <param name="arguments">testhost initialization arguments</param>
+        public static void Run(string arguments)
+        {
+            Task.Run(() =>
+            {
+                Program.Run(arguments.Split(' ').ToArray());
```

Comment:
Assumption that an argument cannot contain a string like: `/someKey="some value"`. Is it correct?

Reply:
Yes, we are passing these arguments, & this is how we are creating dictionary args.